subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -582,7 +582,7 @@ uint8_t ll_adv_aux_ad_data_set(uint8_t handle, uint8_t op, uint8_t frag_pref,
 
 	if (adv->is_enabled) {
 		struct ll_adv_aux_set *aux;
-		struct pdu_adv *pdu;
+		struct pdu_adv *pdu2;
```

Comment:
Suggest to either move existing `pdu` from out of the guard and reuse here, or rename this to `chan_res_pdu`